js/ad_manager_vast.js

```diff
@@ -3448,9 +3448,12 @@ OO.Ads.manager(function(_, $) {
      */
     var _updateCreativeSize = _.bind(function() {
       if (this._slot) {
+        var clientRect = this._slot.getBoundingClientRect();
```

Comment:
jsdom does not support editing offsetWidth/Height, so making use of getBoundingClientRect instead for tests.